extensions/deepcast/src/index.tsx

```diff
@@ -176,8 +180,29 @@ const Command = (props: LaunchProps<{ launchContext?: LaunchContext }>) => {
           <Form.Dropdown.Item value={value} title={title} key={value} />
         ))}
       </Form.Dropdown>
+      {["DE", "FR", "IT", "JA", "PL", "PT-PT", "PT-BR", "RU", "ES"].includes(targetLanguage) && showFormalityConfig && (
```

Comment:
Could this array be assigned a more descriptive constant , so it doesn't seem so random?